build.gradle.kts

```diff
@@ -25,7 +25,7 @@ import org.gradle.gradlebuild.ProjectGroups.pluginProjects
 import org.gradle.gradlebuild.ProjectGroups.publishedProjects
 
 buildscript {
-    project.apply(from = "$rootDir/gradle/shared-with-buildSrc/mirrors.gradle.kts")
+    project.apply(from = "$rootDir/gradle/shared-with-buildSrc/repositories.gradle.kts")
```

Comment:
Are we adding the `jCenter()` repository by default to `buildSrc`? Do some plugins we use add some repositories? If we don't rewrite the URLs of those repositories, then we will still hit them.

Reply:
I didn't find any logic that Gradle "by default" adds `jcenter()`, do you mean AGP adding `jcenter` as default? https://blog.bintray.com/2015/02/09/android-studio-migration-from-maven-central-to-jcenter/